x-pack/legacy/plugins/uptime/server/lib/adapters/monitor_states/elasticsearch_monitor_states_adapter.ts

```diff
@@ -209,74 +209,80 @@ export class ElasticsearchMonitorStatesAdapter implements UMMonitorStatesAdapter
                     state.checksByAgentIdIP = new HashMap();
                 `,
                   map_script: `
-                    Map curCheck = new HashMap();
-                    String agentId = doc["agent.id"][0];
-                    String ip = null;
-                    if (doc["monitor.ip"].length > 0) {
-                      ip = doc["monitor.ip"][0];
-                    }
-                    String agentIdIP = agentId + "-" + (ip == null ? "" : ip.toString());
-                    def ts = doc["@timestamp"][0].toInstant().toEpochMilli();
-                    
-                    def lastCheck = state.checksByAgentIdIP[agentId];
-                    Instant lastTs = lastCheck != null ? lastCheck["@timestamp"] : null;
-                    if (lastTs != null && lastTs > ts) {
-                      return;
-                    }
-                    
-                    curCheck.put("@timestamp", ts);
-                    
-                    Map agent = new HashMap();
-                    agent.id = agentId;
-                    curCheck.put("agent", agent);
-                    
-                    if (state.globals.url == null) {
-                      Map url = new HashMap();
-                      Collection fields = ["full", "original", "scheme", "username", "password", "domain", "port", "path", "query", "fragment"];
-                      for (field in fields) {
-                        String docPath = "url." + field;
-                        def val = doc[docPath];
-                        if (!val.isEmpty()) {
-                          url[field] = val[0];
-                        }
-                      }
-                      state.globals.url = url;
-                    }
-                    
-                    Map monitor = new HashMap();
-                    monitor.status = doc["monitor.status"][0];
-                    monitor.ip = ip;
-                    if (!doc["monitor.name"].isEmpty()) {
-                      String monitorName = doc["monitor.name"][0];
-                      if (monitor.name != "") {
-                        monitor.name = monitorName;
+                  Map curCheck = new HashMap();
```

Comment:
Is there a reason this indentation changed? It's making this diff hard to read.

Reply:
@shahzad31 showed me that you can disable this diffing with this UI widget:
![image](https://user-images.githubusercontent.com/18429259/64036153-84dd5300-cb20-11e9-96c6-aff99a620964.png)

In any case, I checked out the file on this branch and the formatter must've updated it because it's not changing now.

Reply:
++